examples/minimal-mdns/advertiser.cpp

```diff
@@ -33,18 +33,21 @@ enum class AdvertisingMode
 {
     kCommisioning,
     kOperational,
+    kCommisionable,
 };
 
 struct Options
 {
     bool enableIpV4                 = false;
     AdvertisingMode advertisingMode = AdvertisingMode::kCommisioning;
 
-    // commisioning params
+    // commisioning/commisionable params
     uint8_t shortDiscriminator = 52;
     uint16_t longDiscriminator = 840;
     Optional<uint16_t> vendorId;
     Optional<uint16_t> productId;
+    Optional<const char *> pairingInstr;
```

Comment:
Is this required for commisionable as well? if not, I think we should add a comment that these 2 are for commisionable only.

Reply:
Only required for commisionable. Will add a comment.